src/history.rs

```diff
@@ -262,8 +251,10 @@ impl<'a> DynamicHistory<'a> {
         self.increment_local_clock_count();
         self.write_current_clocks_to_log();
         Ok(CausalSnapshot {
-            probe_id: self.probe_id.get_raw(),
-            clock: self.clocks[0].count,
+            clock: LogicalClock {
```

Comment:
This could probably become `clock: self.clocks[0],`

Reply:
:+1: